### PR TITLE
Fix mutations

### PR DESCRIPTION
**Pull request by dgollahon**
Fix the mutations that snuck in while the CI mutation testing config was not operating correctly.

### CODE REVIEW

File: meta/kwoptarg.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```
Comment (backus) [inline meta/kwoptarg.rb]: why were these renamed?

Comment (dgollahon) [inline meta/kwoptarg.rb]: `#hintworthy_node?` checks if the default is a hash. I could have added an entirely new example but since `baz` was an arbitrary placeholder it seemed appropriate to substitute a hash.

tl;dr: it kills the mutation.

Comment (backus) [inline meta/kwoptarg.rb]: Ah. I usually add a new example since otherwise you run the risk of editing a test that kills other mutations

Comment (dgollahon) [inline meta/kwoptarg.rb]: Yeah, I considered that but since `baz` is unaltered it didn't seem like it mattered. I suppose it _could_ though. I am currently doing a full run on the entire codebase on my laptop so I can confirm / fix that if it is a problem later. Or I can add another example if you prefer--it's just more duplication.

Comment (dgollahon) [inline meta/kwoptarg.rb]: We just need to eventually add a way of mapping the specs back onto the subject so it doesn't matter and you don't have to worry about fragile things like that.

Comment (dgollahon) [inline meta/kwoptarg.rb]: Ok, I lied. I forgot you have to do special things to run `mutest` on itself (at least, I assume that's the problem). Otherwise we have a bug.

Comment (dgollahon) [inline meta/kwoptarg.rb]: Incidentally, #34 would have helped since it had already collected a few alives before halting.

Comment (backus) [inline meta/kwoptarg.rb]: 👍 since it seems harmful in this case. In the future lets just add a new example though. It is better than potentially introducing mutations
